wrc/parse/lexer.py

```diff
@@ -108,6 +108,11 @@ def t_PARBREAK(self, token):
         token.lexer.lineno += len(token.value)
         return token
 
+    def t_trailingwhitespace(self, token):
```

Comment:
This token is not meant to reach the parser, it's just here to catch these specific errors.
Without these there is an error but it's totally unclear what the hell is wrong...
I should implement more of these to fully fix #4!